### PR TITLE
Corrections for 01:00 to 01:15

### PR DESCRIPTION
**Pull request by edwardsmit**

### CODE REVIEW

**Comment by nelstrom** [comment]
Excellent work @edwardsmit, thanks!

File: 3/ollie-charles-captions.srt [hunk 1]
```diff
@@ -4377,10 +4378,13 @@ We'll just take the word chains out of that.
...
 01:04:54,770 --> 01:04:59,050
-(inaudible)
+SP: Just the first of head
+OC: Oh yes
 
 1003
 01:05:09,420 --> 01:05:11,691
+OC: So maybe,... rather than the head
+SP: Now it's map first
 So every time we take one more
```
Comment (nelstrom) [inline 3/ollie-charles-captions.srt:4388]: Prefix "OC: " in front of "So every time we take one more"

File: 3/ollie-charles-captions.srt [hunk 2]
```diff
@@ -4440,7 +4444,7 @@ so if we take the original pairs and what we
 
 1017
 01:05:57,331 --> 01:06:02,450
-need to do is go again on the concat map.
+need to do is go again on the concatMap.
```
Comment (nelstrom) [inline 3/ollie-charles-captions.srt:4447]: :+1: good catch!

File: 3/ollie-charles-captions.srt [hunk 3]
```diff
@@ -4773,11 +4777,11 @@ To me, the biggest source of expense
 
 1095
 01:10:15,250 --> 01:10:16,902
-feels like this neighbors function.
+feels like this neighbours function.
```
Comment (nelstrom) [inline 3/ollie-charles-captions.srt:4780]: I was about to say "No, we're preferring American spellings to UK spellings", but then remembered that this is the name of a function Ollie wrote, so we'll copy his preferred spelling, which is British.

File: 3/ollie-charles-captions.srt [hunk 4]
```diff
@@ -4913,7 +4917,7 @@ And this becomes straightforward,
 
 1129
 01:11:54,370 --> 01:12:01,080
-we simply do a map look-up (inaudible)
+we simply do a map look-up which is the exclamation operator
```
Comment (nelstrom) [inline 3/ollie-charles-captions.srt:4920]: Nice one.

File: 3/ollie-charles-captions.srt [hunk 5]
```diff
@@ -4936,7 +4940,7 @@ OC: If I was doing this production code
 1134
 01:12:12,960 --> 01:12:14,291
 I would probably want
-to keep the maybes' 
+to keep the maybes'
```
Comment (nelstrom) [inline 3/ollie-charles-captions.srt:4943]: I don't think that apostrophe is needed.

File: 3/ollie-charles-captions.srt (hunk 3 above)
Comment (edwardsmit) [inline 3/ollie-charles-captions.srt:4780]: Yes, I've tried to only change the neighbors to neighbours when he discusses the function explicitly. They als talk a lot about the concept of neighbors and I've tried to leave those in American spelling

**Pull request merged**